lib/read/lib/Cardano/Wallet/Read/Block/HeaderHash.hs

```diff
@@ -46,15 +51,24 @@ import Cardano.Wallet.Read.Eras.KnownEras
     ( Era (..)
     , IsEra (..)
     )
+import Cardano.Wallet.Read.Hash
+    ( Blake2b_256
```

Comment:
this one could be exported from `cryptography-primitive` lib

Reply:
(See the other comment.)